.editorconfig

```diff
@@ -0,0 +1,14 @@
+root = true
+
+[*]
+charset = utf-8
+end_of_line = lf
+trim_trailing_whitespace = true
+insert_final_newline = true
+
+[*.{rs,toml}]
+indent_style = space
+indent_size = 4
+
+[*.md]
+max_line_length = 80
```

Comment:
I picked 80 to match the limit currently being enforced on `extensions/**/*.md` by `deno fmt`.